builtin/common/information_formspecs.lua

```diff
@@ -110,6 +119,98 @@ local function build_privs_formspec(name)
 end
 
 
+-- Checks whether the mod with the given name is a separate mod or a part of a modpack
+local function is_mod_sep_or_part_of_mp(modname)
+	local modpath = core.get_modpath(modname)
+	local find_ptn = "[^" .. DIR_DELIM .. "]*.$"
+	local modpart = modpath:match(find_ptn)
+	local mp_part = modpath:sub(1, modpath:len()-modpart:len()-1):match(find_ptn)
```

Comment:
And I have no idea how this works either

Reply:
I've rewritten the code in the new commit.